coopersystem-fsd/pcc-framework
# Fixed white screen on editor pages

## Description

- Added 'core/list-item' to the white list of blocks

## Steps to test

1. Edit a post of any type or create a new one and insert a text in paragraph block

**Expected behavior: If the post is edited, the screen will not go blank; if the text is entered in the editor, a dialog with an error message will not be displayed.

## Additional information
